modules/aws_ecs_fargate/security.tf

```diff
@@ -0,0 +1,54 @@
+resource "aws_security_group" "rds" {
+  name        = "${var.deployment_name}-rds-security-group"
+  description = "Retool database security group"
+
+  ingress {
+    description = "Retool ECS Postgres Inbound"
+    from_port   = "5432"
+    to_port     = "5432"
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
this should probably be a var too - i doubt people would by default want everything to be able to reach their rds instance